resources/views/welcome.blade.php

```diff
@@ -1,138 +1,117 @@
 <!DOCTYPE html>
-<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
-<head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
+<html>
+<main>
```

Comment:
`<main>` is not a part of an HTML document. It normally should be `<html>`  with only 2 children: `<head>` and `<body>`.